src/Message/Content/Image.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+declare(strict_types=1);
+
+namespace PhpLlm\LlmChain\Message\Content;
+
+final readonly class Image implements ContentInterface
+{
+    public function __construct(public string $image)
```

Comment:
```suggestion
    public function __construct(public string $url)
```

And maybe also assert it starts with `http` in the actor? Or too much?

Reply:
too much, data urls are also valid

Reply:
e.g. something like this is also valid:
```json
        {
          "type": "image_url",
          "image_url": {
            "url": f"data:image/jpeg;base64,{base64_image}"
          }
        }
```